micro-infra-spring-config/src/test/groovy/com/ofg/infrastructure/property/FileSystemPollerTest.groovy

```diff
@@ -39,23 +43,42 @@ class FileSystemPollerTest extends AbstractIntegrationTest {
         poller = context.getBean(FileSystemPoller)
     }
 
-    def 'should reload configuration once when file system touched'() {
+    @Unroll
+    def 'should reload configuration once when file #configFile from config location with allowed name is touched'() {
         when:
-            oneConfigurationFileWasChanged("micro-app.yaml")
+            oneConfigurationFileWasChanged(configFile)
+
         then:
-            conditions.eventually {
-                counter.value == old(counter.value) + 1
-            }
-            counter.value == old(counter.value) + 1
+            int oldValue = old(counter.value)
+            waitForCounterIncrement(oldValue)
+            assertThatCounterWasIncrementedOnce(oldValue)
+
+        where:
+            configFile << [
+                    poller.getConfigLocations().commonPropertiesFile(MICROSERVICE_NAME),
+                    poller.getConfigLocations().commonYamlFile(MICROSERVICE_NAME),
+                    poller.getConfigLocations().envPropertiesFile(MICROSERVICE_NAME),
+                    poller.getConfigLocations().envYamlFile(MICROSERVICE_NAME),
+                    poller.getConfigLocations().countryPropertiesFile(PL_MICROSERVICE_NAME),
+                    poller.getConfigLocations().countryYamlFile(PL_MICROSERVICE_NAME)
+            ]
     }
 
-    private void oneConfigurationFileWasChanged(String configFile) {
-        File file = new File(poller.getConfigPath(), configFile)
-        assert file.exists()
-        file.setLastModified(System.currentTimeMillis())
+    private void waitForCounterIncrement(int oldValue) {
+        conditions.eventually {
+            counter.value == oldValue + 1
+        }
     }
-}
 
+    private boolean assertThatCounterWasIncrementedOnce(int oldValue) {
+        return counter.value == oldValue + 1
```

Comment:
It would probably returns "true != false" error message instead of `1 != 2` (it would be better to use `assert` keyword in a method).